Vault-Deployment/values/vault-values.yaml

```diff
@@ -11,9 +11,7 @@ server:
   dataStorage:
     enabled: true
     size: 1Gi
-  auditStorage:
```

Comment:
@eyenx @pree I think somewhere here you might be able to hook up an `emptyDir`, haven't checked it yet

Reply:
https://github.com/adfinis/vault-playground-on-kubernetes/issues/28